assets/javascript/app.js

```diff
@@ -7,12 +7,13 @@ $(document).ready(function() {
     var lat;
     var long;
     var brewer;
+    var filter = false
```

Comment:
Set up a switch to control the filter box behavior when searching for a new location. Switch starts in false. If search button is used it flips this switch and runs through localbrews(). This function has logic at the top (L262-269) that says is the switch it on, then hide the filter box if there are results, otherwise keep it open so the user can search for another location. This then resets the switch for later searches.